app.js

```diff
@@ -3,16 +3,15 @@ require('dotenv').config();
 const express = require('express');
 const hbs = require('hbs');
 
-// require spotify-web-api-node package here:
-
 const app = express();
 
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
-// setting the spotify-api goes here:
 
 // Our routes go here:
+const router = require('./config/routes.config');
+app.use(router);
 
 app.listen(3000, () => console.log('My Spotify project running on port 3000 🎧 🥁 🎸 🔊'));
```

Comment:
```suggestion
const port = process.env.PORT || 3000
app.listen(port, () => console.log(`My Spotify project running on port ${port} 🎧 🥁 🎸 🔊`));
```

Un pequeño cambio de config a ver si te mola! :D